src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/SQLCurationDao.scala

```diff
@@ -106,7 +108,7 @@ class SQLCurationDao(implicit connection: Connection, errorHandler: SQLErrorHand
         assert(ts.nonEmpty)
         ts.collectFirst {
           case t if errorHandler.isForeignKeyError(t) => NoSuchDepositError(id)
-          case t if errorHandler.isUniquenessConstraintError(t) => DepositIdAndTimestampAlreadyExistError(id, curation.timestamp, "curation")
+          case t if errorHandler.isUniquenessConstraintError(t) => DepositIdAndTimestampAlreadyExistError(id, curation.timestamp, daoName)
```

Comment:
the use of `daoName` here and everywhere in other `SQLxxxDao` on similar places must be with lowercase to make the constructed error message correct. Either use `.toLowercase` in all usages or apply this function in the constructor of `DepositIdAndTimestampAlreadyExistError` itself.